app/controllers/posts_controller.rb

```diff
@@ -1,9 +1,13 @@
 class PostsController < ApplicationController
   def index
     @posts = Post.all
+    @user = User.find_by(id: params[:user_id])
+    @user_posts = @posts.where(author_id: params[:user_id])
   end
 
   def show
-    @post = 'post'
+    @post = Post.find_by(id: params[:id]) || 'No posts yet'
+    @post_comments = Comment.all.where(post_id: params[:id])
+    @user = User.find_by(id: @post.author_id)
```

Comment:
- [ ] According to the requirements:

![image](https://github.com/Reggeditt/blog-rails/assets/77069011/2c1dca57-3389-478c-a502-435593b2d0e8)

Please use the `#recent_comments` method you created in your `Post` model for this view. 🤓 🔧

Reply:
fixed